plaso/engine/profilers.py

```diff
@@ -43,12 +44,14 @@ def SampleStop(self):
 class SampleFileProfiler(object):
   """Shared functionality for sample file-based profilers."""
 
-  _FILENAME_PREFIX = None
+  _FILENAME_PREFIX = ''
```

Comment:
I opt to change this back to None and add a comment that this value should be overwritten by a subclass.

Reply:
Done, though the assigning to protected constants in the test doesn't feel good.